common/test/unit/com/thoughtworks/go/buildsession/BuildSessionCancelingTest.java

```diff
@@ -76,7 +76,7 @@ public void cancelLongRunningTestCommand() throws InterruptedException {
             @Override
             public void run() {
                 buildSession.build(compose(
-                        echo("after sleep").setTest(execSleepScript(50))));
+                        cond(or(execSleepScript(50), execSleepScript(50)), echo("after sleep"))));
```

Comment:
to make sure testing command after testing command being canceled will not execute.